core/src/toga/widgets/base.py

```diff
@@ -9,22 +9,25 @@
 
 from toga.platform import get_platform_factory
 from toga.style import Pack, TogaApplicator
+from toga.style.mixin import style_mixin
 
 if TYPE_CHECKING:
     from toga.app import App
     from toga.window import Window
 
 StyleT = TypeVar("StyleT", bound=BaseStyle)
+PackMixin = style_mixin(Pack)
 
 
-class Widget(Node):
+class Widget(Node, PackMixin):
     _MIN_WIDTH = 100
     _MIN_HEIGHT = 100
 
     def __init__(
         self,
         id: str | None = None,
         style: StyleT | None = None,
+        **kwargs,
```

Comment:
Although this argument is currently only used for styles, I called it `kwargs` rather than anything style-specific, because at some point I’d like to have a larger cleanup of constructor arguments in which non-style properties could be included. I'll write this up in a separate issue.